### PR TITLE
Add support for Lettera free passes

### PR DESCRIPTION
**Pull request by jllang**

### CODE REVIEW

File: packages/components/src/Lettera.purs [hunk 1]
```diff
@@ -109,19 +109,39 @@ handleLetteraError = Console.warn <<< show
...
-getArticleWithUrl :: String -> Maybe Paper -> Maybe UserAuth -> Maybe String -> Aff (Either String FullArticle)
-getArticleWithUrl url paper auth clientip = do
+getArticleWithUrl
+  :: String
+  -> Maybe Paper
+  -> Maybe UserAuth
+  -> Maybe String
+  -> Maybe String
+  -> Aff (Either String FullArticle)
+getArticleWithUrl url paper auth clientip freePassHash= do
```
Comment (kaol) [inline packages/components/src/Lettera.purs:140]: Add white space.

File: packages/components/src/Lettera.purs [hunk 2]
```diff
@@ -109,19 +109,39 @@ handleLetteraError = Console.warn <<< show
...
   let request = AX.defaultRequest
-        { url = url <> foldMap (\p -> "?paper=" <> Paper.toString p) paper
+        { url = url <>
+                foldMap (\p -> "?paper=" <> Paper.toString p) paper <>
+                foldMap (\f -> "&freePassHash=" <> f) freePassHash
```
Comment (kaol) [inline packages/components/src/Lettera.purs:144]: Is the token guaranteed to only have URL safe characters?

**Review by kaol — APPROVED** [review]

File: packages/components/src/Lettera.purs (hunk 2 above)
Comment (jllang) [reply packages/components/src/Lettera.purs:144]: The token comes from URL, but I can add sanitation.

Comment (jllang) [reply packages/components/src/Lettera.purs:144]: Is it okay if I sanitize free pass hash already when parsing the URL in Mosaico? I could add a comment that the hash should be sanitized by the caller of  this function.

**Pull request merged**